hooks/useStore.js

```diff
@@ -30,6 +30,22 @@ const config = (set) => {
       });
     },
 
+    updateTimer: (editProject) => {
+      set((draft) => {
+        const draftProjectArr = draft.projects.map((project) => {
```

Comment:
Does this mean `draftProjectArray` or `draftProjectArrow`? I think that there is no need to abbreviate here and instead be specific about what's going on.

Reply:
it means draftProjectArray, i map over the projects and store the result in the draftProjectArray. for me it was clear cause the map function, that results an array. but i have changed it for better readability.